### DISCUSSION TITLE
Your recreations

### DISCUSSION THREAD

**Discussion by nasko222** [nasko222/the-minecraft-recreation-project#170] [Show and tell]
Yall also want to make recreations I got a few mails about it, post your download links for remakes here or source codes etc

## and remember that anything here is from third party and wont be verified by The Minecraft Recreation Project, so download only if you trust it.

**Comment by 483aa** [comment]


**Reply by RossGamer2048** [reply]
where's pc-9? its created on may 9 2009?

**Comment by 483aa** [comment]
[CGTT versions with new cave gen.zip](https://github.com/nasko222/the-minecraft-recreation-project/files/14738797/CGTT.versions.with.new.cave.gen.zip)
pc-131951 in development.
It will be added to the archive in the future

**Reply by 483aa** [reply]
pc-131951 it will remain with old cave gen

**Comment by TechGuysTechSpotOfficial** [comment]
here's my remakes, pretty crappy as expected since I suck at java
[my remakes.zip](https://github.com/nasko222/the-minecraft-recreation-project/files/14739482/my.remakes.zip)

**Reply by TechGuysTechSpotOfficial** [reply]
This doesn't include my remakes that I am currently working on:
Minecraft Live 2020 Remakes

**Reply by Illia24** [reply]
Wait mc live 2020 recreations? Man im starving for those, remind me when you finish those pls

**Reply by TechGuysTechSpotOfficial** [reply]
they are just retextured snapshots but they kinda do the job, I've been busy lately so I haven't uploaded them yet

**Reply by Illia24** [reply]
still will be waiting

**Reply by Spicytomatoes777** [reply]
hes still waiting lol

**Reply by Illia24** [reply]
Nah

**Comment by Spicytomatoes777** [comment]
speculative remakes (in-20100129-dev and 0.0.2a-dev)
[Remakes](https://github.com/user-attachments/files/21567987/Remakes.zip)

**Reply by lol-AX** [reply]
What's about 0.0.2a-dev ?

**Reply by Spicytomatoes777** [reply]
wdym
